src/tools/annotation/bidirectionalTool/moveHandle/getBaseData.test.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-eval */
+import getBaseData from './getBaseData';
+
+jest.mock('./../../../../externalModules.js', () => {
+  const intersectLine = eval(
```

Comment:
I'm not sure I understand what's going on here. Why the `eval`?

Reply:
I was mocking the original implementation of intersectLine from `cornerstoneMath`.
Just changed that and started mocking the returned value for the test instead.
Thank you!